src/pages/create-user-page.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { CreateUserForm } from "../components/mol.create-user-form/create-user-form.component";
-import { useCreateUser } from "../domain/create-user.use-case";
+import { useCreateUser } from "../domain/create-user-use-case";
```

Comment:
geralmente usamos a convenção `nome-do-arquivo.tipo-do-arquivo`
```suggestion
import { useCreateUser } from "../domain/create-user.use-case";
```

Reply:
https://www.notion.so/taqtile/Conven-es-para-guias-de-estilo-v-1-0-e116bcbe61ee41ddb1a3892781186e14?pvs=4 

estou estudando novamente nossa convenção.